UserService/app/src/main/java/com/bookmyshow/userservice/dto/RoleResponseDTO.java

```diff
@@ -0,0 +1,59 @@
+package com.bookmyshow.userservice.dto;
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonPropertyOrder({
+        "id",
+        "role",
+        "roleDescription"
+})
+public class RoleResponseDTO {
+
+    private Long id;
+
+    private String role;
+
+    private String roleDescription;
+
+    public RoleResponseDTO() {
+    }
+
+    public RoleResponseDTO(Long id, String role, String roleDescription) {
+        this.id = id;
+        this.role = role;
+        this.roleDescription = roleDescription;
+    }
+
+    @Override
+    public String toString() {
+        return "RoleDTO{" +
+                "id=" + id +
+                ", role='" + role + '\'' +
+                ", roleDescription='" + roleDescription + '\'' +
+                '}';
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getRole() {
+        return role;
+    }
+
+    public void setRole(String role) {
+        this.role = role;
+    }
+
+    public String getRoleDescription() {
+        return roleDescription;
+    }
```

Comment:
we can include Lombok dependency and use the custom getter/setters as needed in respective classes